src/color.rs

```diff
@@ -64,12 +99,44 @@ impl Color {
                     code = ansi256_from_rgb((*r, *g, *b))
                 ),
             },
+            Color::Ansi3Bit(color) => format!("{}", *color as u8 + colortype.bg_addition()),
         }
     }
+
+    fn from_ansi_name(s: &str) -> Result<Color> {
+        match s {
+            "black" => Ok(Self::Ansi3Bit(Ansi3Bit::Black)),
+            "red" => Ok(Self::Ansi3Bit(Ansi3Bit::Red)),
+            "green" => Ok(Self::Ansi3Bit(Ansi3Bit::Green)),
+            "yellow" => Ok(Self::Ansi3Bit(Ansi3Bit::Yellow)),
+            "blue" => Ok(Self::Ansi3Bit(Ansi3Bit::Blue)),
+            "magenta" => Ok(Self::Ansi3Bit(Ansi3Bit::Magenta)),
+            "cyan" => Ok(Self::Ansi3Bit(Ansi3Bit::Cyan)),
+            "white" => Ok(Self::Ansi3Bit(Ansi3Bit::White)),
+            "bright_black" => Ok(Self::Ansi3Bit(Ansi3Bit::BrightBlack)),
+            "bright_red" => Ok(Self::Ansi3Bit(Ansi3Bit::BrightRed)),
+            "bright_green" => Ok(Self::Ansi3Bit(Ansi3Bit::BrightGreen)),
+            "bright_yellow" => Ok(Self::Ansi3Bit(Ansi3Bit::BrightYellow)),
+            "bright_blue" => Ok(Self::Ansi3Bit(Ansi3Bit::BrightBlue)),
+            "bright_magenta" => Ok(Self::Ansi3Bit(Ansi3Bit::BrightMagenta)),
+            "bright_cyan" => Ok(Self::Ansi3Bit(Ansi3Bit::BrightCyan)),
+            "bright_white" => Ok(Self::Ansi3Bit(Ansi3Bit::BrightWhite)),
```

Comment:
Wondering if we should prefix those with `ansi:` or `ansi_` or similar? Otherwise, someone seeing "red" in a theme file might expect the color to actually be `#ff0000` (like if you would specify "red" in a CSS file), while in reality, it would depend on the terminal emulator settings.

And this would allow us to introduce CSS-style or X11-style color names in the future.

Reply:
Okay, I have updated it to use the prefix `ansi:`.  I personally would be against using CSS or X11 colour names because, unlike the ansi ones, do not correspond to something variable, and so could just be written as their respective colours in the definitions at the top of the file.

Reply:
> I personally would be against using CSS or X11 colour names because, unlike the ansi ones, do not correspond to something variable, and so could just be written as their respective colours in the definitions at the top of the file.

You are right